Task_4/admin-panel/src/app/layout.tsx

```diff
@@ -18,15 +19,17 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={inter.className}>
-        <div className="flex">
-          <Vnavbar />
-          <div className="w-full">
-            <Navbar />
-            {children}
+      <StoreProvider>
+        <body className={inter.className}>
+          <div className="flex">
+            <Vnavbar />
```

Comment:
@codal-uchoudhary why `Vnavbar`? Shouldn't it be only `Navbar`...

Reply:
Navbar and v-navbar (vertical navbar ) are different component.